pyflakes/checker.py

```diff
@@ -1511,6 +1519,33 @@ def SUBSCRIPT(self, node):
                 self.handleChildren(node)
             finally:
                 self._in_typing_literal = orig
+        elif (
+                (
+                    isinstance(node.value, ast.Name) and
+                    node.value.id == 'Annotated'
+                ) or (
+                    isinstance(node.value, ast.Attribute) and
+                    node.value.attr == 'Annotated'
+                )
+        ):
+            self.handleNode(node.value, node)
+
+            if (
+                    isinstance(node.slice, ast.Index) and
+                    isinstance(node.slice.value, ast.Tuple)
+            ):
```

Comment:
this if block ends up being not-quite-correct in python3.9+  -- the `ast.Index` type is gone and this is just a `Tuple` directly (this leads to one test failure on python3.9 -- let me know if you want me to try and hack on this for ya since it's a bit tricky)

Reply:
Thanks @asottile for the heads up. If you could work on this that'd be great.